index.js

```diff
@@ -149,24 +149,41 @@ class ServerlessCustomDomain {
       let nameLength = 0;
       // The arn of the choosen certificate
       let certificateArn;
+      // The certificate name
+      let certificateName = this.serverless.service.custom.customDomain.certificateName;
+
+      // Checks if a certificate name is given
+      if (certificateName.trim() !== '') {
+        data.CertificateSummaryList.some((certificate) => {
+          if (certificate.DomainName === certificateName) {
+            certificateArn = certificate.CertificateArn;
```

Comment:
Modifying a outer variable from here seems a bit error prone. Why not use something like `filter` to get back the correct certificate and pull out its Arn if its found